go-controller/pkg/types/const.go

```diff
@@ -13,6 +13,10 @@ const (
 	// access to local service
 	LocalNetworkName = "locnet"
 
+	// OvnServiceIdledAt is a constant string representing the Service annotation key
+	// whose value indicates the time stamp in RFC3339 format when a Service was idled
+	OvnServiceIdledAt = "k8s.ovn.org/idled-at"
```

Comment:
nit: change OvnServiceIdledAt -> OVNServiceIdledAt, We should be moving towards keeping initialisms the same case per the go docs 

https://github.com/golang/go/wiki/CodeReviewComments#initialisms

Reply:
done, thanks for the link